index.css

```diff
@@ -1,466 +1,709 @@
+  margin-bottom: 3%;
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+}
+
+footer form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+  padding: 5%;
+  gap: 1em;
+}
+
+footer form .text-inputs {
+  border-radius: 8px;
+  border-color: #cfd8dc;
+  line-height: 2em;
+  width: 327px;
+  font-size: large;
+  padding: 5px;
+}
+
+footer form .message-input {
+  height: 170px;
+  border-radius: 8px;
+  border-color: #cfd8dc;
+  width: 327px;
+  font-size: large;
+  padding: 5px;
+  background-color: #fff;
+  left: 16px;
+  right: 38px;
+  top: 15px;
+  bottom: 15px;
+}
+
+@media (min-width: 768px) {
   body {
-    background-color: #dfe1e6;
-    overflow-x: hidden;
-    margin: 0 auto;
-    max-width: 375px;
-    height: auto;
-  }
-  
-  .d-flex {
-    display: flex;
+    background: #e5e5e5;
   }
-  
-  nav {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    padding: 10px 24px 10px 24px;
-    background-color: #fff;
+
+  .d-text {
+    color: #091e42;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    width: 100%;
+    margin-left: 18%;
   }
-  
+
+  .bio .d-text {
+    width: 64%;
+  }
+
   .logo {
-    text-decoration: none;
-    color: var(--calm-purple);
-    width: 76px;
-    height: 20px;
-    font-family: "Poppins", sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 18px;
-    line-height: 20px;
-    display: flex;
-    align-items: center;
-    text-align: center;
+    margin-left: 30px;
   }
-  
+
   .hamburger {
-    color: var(--calm-purple);
-    width: 24px;
-    height: 24px;
+    display: none;
   }
-  
-  .main-container {
-    margin-top: 0;
-    background-color: #f7f7f9;
-  }
-  
+
   .intro {
-    background-image: url(./assets/hero-background.svg);
-    background-size: cover;
-    height: 604px;
-    border-bottom-left-radius: 70px;
+    background: url("assets/Header-bg.png");
+    border-bottom-left-radius: 60px;
+    background-repeat: no-repeat;
+    height: 916px;
+    width: 100%;
     background-color: #fff;
   }
-  
-  .intro-text {
+
+  .greeting.h-text1 {
+    margin-top: 6%;
+    width: calc(100% - 36.11px);
+    margin-left: 18%;
+    margin-right: 18%;
+  }
+
+  .overall-navi {
+    position: fixed;
+    width: 100%;
     display: flex;
-    flex-direction: column;
-    padding: 114px 24px 0 24px;
+    justify-content: space-between;
   }
-  
-  .greeting {
-    height: 150px;
-    font-family: "Poppins", sans-serif;
-    font-style: normal;
-    font-weight: 700;
-    font-size: 40px;
-    line-height: 52px;
+
+  .desktop-navi {
     display: flex;
+    flex-direction: row;
     align-items: center;
-    margin-bottom: 0;
-  }
-  
-  .bio {
-    margin-top: 13px;
-    width: 95%;
-    max-width: 640px;
+    text-align: center;
+    padding: 0;
+    gap: 8px;
```

Comment:
Good job with adding desktop navigation 👍  But let's make sure we follow the correct space between the links like below. 👍 
<img width="617" alt="Screenshot 2023-02-25 at 9 02 07 PM" src="https://user-images.githubusercontent.com/25887253/221373184-cdaebffc-843a-4fc3-9068-0c369cffa4d4.png">
<img width="428" alt="Screenshot 2023-02-25 at 9 17 41 PM" src="https://user-images.githubusercontent.com/25887253/221373201-efb3dfa7-077b-4622-be8c-4dbc0f197b09.png">